pkgs/development/tools/frink/default.nix

```diff
@@ -0,0 +1,35 @@
+{ pkgs }:
+with pkgs;
+let
+  jar = fetchurl {
+    url = "https://frinklang.org/frinkjar/frink.jar";
```

Comment:
It does not have any version string?

Reply:
Nope, not that I could find :(
I will contact the author to ask if there are maybe undocumented versioned jars around somewhere.

Reply:
Nope, not that I could find :(
I will contact the author to ask if there are maybe undocumented versioned jars around somewhere.

Reply:
How the author announces new releases?